src/config/config.c

```diff
@@ -1018,6 +1018,12 @@ void initConfig(struct config *conf)
 	conf->webserver.api.app_pwhash.d.s = (char*)"";
 	conf->webserver.api.app_pwhash.c = validate_stub; // Only type-based checking
 
+	conf->webserver.api.app_sudo.k = "webserver.api.app_sudo";
+	conf->webserver.api.app_sudo.h = "Should application password API sessions be allowed to modify config settings?\n Setting this to true allows third-party applications using the application password to modify advanced settings, e.g., the upstream DNS servers, DHCP server settings, or changing passwords. This setting should only be enabled if really needed and only if you trust the applications using the application password.";
```

Comment:
I think this affects all settings, not only advanced settings.

```suggestion
	conf->webserver.api.app_sudo.h = "Should application password API sessions be allowed to modify config settings?\n Setting this to true allows third-party applications using the application password to modify settings, e.g., the upstream DNS servers, DHCP server settings, or changing passwords. This setting should only be enabled if really needed and only if you trust the applications using the application password.";
```

Reply:
We also need to edit `test/pihole.toml` or CI tests will complain that the code does not match the config file. I will, hence, do the changed locally and push them instead of simply accepting the suggestion here.